.github/workflows/test_iasworld_data.yaml

```diff
@@ -0,0 +1,141 @@
+name: test-iasworld-data
```

Comment:
This workflow should be nearly identical to [the old implementation of `test-dbt-models`](https://github.com/ccao-data/data-architecture/blob/38b066ed2b4e1c03f21bd6a8ef9d249197e65518/.github/workflows/test_dbt_models.yaml#L1-L146), but GitHub can't pick up on the diff because we're keeping `test-dbt-models` and adapting it for a slightly more specific purpose. I'll call out every change I made in comments below.